ols/src/query_helpers/docs_summarizer.py

```diff
@@ -92,7 +93,22 @@ def summarize(
 
         logger.info(f"{conversation_id} Getting service context")
 
-        embed_model = "local:BAAI/bge-base-en"
+        embed_model: Optional[Any]  # need this to make mypy happy
+        if (
+            config.ols_config.reference_content is not None
+            and config.ols_config.reference_content.embeddings_model_path is not None
+        ):
+            from langchain_community.embeddings import HuggingFaceBgeEmbeddings
+
+            os.environ["TRANSFORMERS_CACHE"] = (
+                config.ols_config.reference_content.embeddings_model_path
+            )
+            os.environ["TRANSFORMERS_OFFLINE"] = "1"
+            embed_model = HuggingFaceBgeEmbeddings(
+                model_name=config.ols_config.reference_content.embeddings_model_path
+            )
+        else:
+            embed_model = "local:BAAI/bge-base-en"
```

Comment:
so if we end up here we'll attempt to download the model?

Reply:
Yes, if there's no embeddings_model_path in config, the behavior is as before.